cocos/core/value-types/bitmask.ts

```diff
@@ -107,16 +107,35 @@ BitMask.getList = (BitMaskDef) => {
         return BitMaskDef.__bitmask__;
     }
 
-    const bitlist: any[] = BitMaskDef.__bitmask__ = [];
+    return BitMask.update(BitMaskDef);
+};
+
+/**
+ * @en
+ * Update BitMask object properties.
+ * @zh
+ * 更新 BitMask 对象的属性列表。
+ *
+ * @param BitMaskDef
+ * @returns @en A sorted array with integer values. @zh 存储整型属性值的数组。该数组已排序。
+ */
+BitMask.update = (BitMaskDef) => {

```

Comment:
Is this applying a bitmask to the current bitmask? If so, would the name "apply" be more appropriate?
The function appears to update the bitmask. If the bitmask can be modified, why not provide interfaces such as "add"?